<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IoLineWidget</class>
 <widget class="QWidget" name="IoLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_type"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_name"/>
   </item>
   <item>
    <widget class="QToolButton" name="m_remove">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
